docs/UsersGuide/source/Quickstart.rst

```diff
@@ -1,320 +1,325 @@
-CONUS grid (RRFS_CONUS_25km), the GFS version 15.2 physics suite (FV3_GFS_v15p2 CCPP), and
-FV3-based GFS raw external model data for initialization.
+.. _QuickstartC:
 
-.. note::
+====================================
+Quick Start Guide
+====================================
 
-   The steps described in this chapter are applicable to preconfigured (Level 1) machines where
-   all of the required libraries for building community releases of UFS models and applications
-   are available in a central place (i.e. the bundled libraries (NCEPLIBS) and third-party
-   libraries (NCEPLIBS-external) have both been built).  The Level 1 platforms are listed `here
-   <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_.
-   For more information on compiling NCEPLIBS-external and NCEPLIBS, please refer to the
-   NCEPLIBS-external `wiki <https://github.com/NOAA-EMC/NCEPLIBS-external/wiki>`_. 
+This Quick Start Guide will help users to build and run the "out-of-the-box" case for the Unified Forecast System (:term:`UFS`) Short-Range Weather (SRW) Application using a :term:`container`. The container approach provides a uniform enviroment in which to build and run the SRW App. Normally, the details of building and running the SRW App vary from system to system due to the many possible combinations of operating systems, compilers, :term:`MPI`’s, and package versions available. Installation via an :term:`EPIC`-provided container reduces this variability and allows for a smoother SRW App build and run experience. However, the :ref:`non-container approach <BuildRunSRW>` may be more appropriate for those users who desire additional customizability, particularly if they already have experience running the SRW App. 
 
+The "out-of-the-box" SRW App case described in this User's Guide builds a weather forecast for June 15-16, 2019. Multiple convective weather events during these two days produced over 200 filtered storm reports. Severe weather was clustered in two areas: the Upper Midwest through the Ohio Valley and the Southern Great Plains. This forecast uses a predefined 25-km Continental United States (:term:`CONUS`) grid (RRFS_CONUS_25km), the Global Forecast System (:term:`GFS`) version 15.2 physics suite (FV3_GFS_v15p2 :term:`CCPP`), and :term:`FV3`-based GFS raw external model data for initialization.
 
-Download the UFS SRW Application Code
-=====================================
-The necessary source code is publicly available on GitHub.  To clone the release branch of the repository:
+.. attention::
 
-.. code-block:: console
+   All UFS applications support `four platform levels <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_. The steps described in this chapter will work most smoothly on preconfigured (Level 1) systems. However, this guide can serve as a starting point for running the SRW App on other systems, too. 
```

Comment:
I believe so because Level 3 & 4 platforms are still going to have issues running the SRW App for lack of compute power, and there are sections of the instructions that note what to do specifically for Level 1 systems (e.g., where to find data).